app/views/layouts/formio.html.haml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+%html
+  %head
+    %meta{ "http-equiv" => "content-type", "content" => "text/html; charset=UTF-8" }
+    :plain
```

Comment:
This mixing of haml and straight html is kind of nasty. Maybe in a future refactor we can either convert this to either full haml or this layout is basic enough I'd be fine with plain `html.erb`.